content/general/concepts/modulo/modulo.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Modulo'
+Description: 'A modulo operation takes two arguments and returns the remainder when the first argument is divided by the second.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arithmetic'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Modulo is a mathematical operation available in every programming language along with more common ones like addition and multiplication. A modulo operation takes two arguments and returns the remainder when the first argument is divided by the second. The result of the operation is called the modulus.
+
+Most programming languages use the symbol `%` to represent the modulo operation, though some may use `mod`.
+
+## Examples
```

Comment:
I think we should combine all the examples below this code block and pick an example programming language. 

```codebyte/js
let modulus = 14 % 5
console.log(`Results in a modulus of ${modulus}`)

modulus = 37 % 3
console.log(`Results in a modulus of ${modulus}`)

modulus = 21 % 7
console.log(`Results in a modulus of ${modulus}`)
```

Let me know what you think! Also wouldn't hurt to get a second opinion.